client/css/style.css

```diff
@@ -1643,11 +1643,16 @@ part/quit messages where we don't load previews (adds a blank line otherwise) */
 	opacity: 1;
 }
 
-.textcomplete-item .emoji {
-	margin-right: 8px;
-	width: 16px;
-	text-align: center;
+.inline-emoji {
 	display: inline-block;
+	font-size: 1.4em;
```

Comment:
Could you actually go a little higher than that? I like the size of the autocomplete emoji, but the ones in the text are still a little too small IMO. Between `1.6` and `1.8em` would be sweet (I'm talking about default theme with no custom styling, btw), but I haven't looked at alignment stuff.

Reply:
I'd be okay if that means incrementing the line height a little bit. I think it's worth it.

Reply:
I don't know, our line-height of 1.4 is already kinda pushing it.

Reply:
Oh well, 1.4 is better than nothing I guess.